atomic_reactor/util.py

```diff
@@ -862,3 +866,49 @@ def are_plugins_in_order(plugins_conf, *plugins_names):
         except ValueError:
             return False
     return True
+
+
+def read_yaml(yaml_file_path, default=None, schema=None):
+    with open(yaml_file_path) as f:
+        data = yaml.safe_load(f)
+
+    if not schema:
+        return data
```

Comment:
Actually, we don't have a use case for that. I'm going to make schema required to encourage always using a schema :)

Also noticed that default parameter is not used. This is leftover from a previous version. Removing it as well.